src/backend/distributed/utils/role.c

```diff
@@ -96,11 +96,11 @@ worker_create_or_alter_role(PG_FUNCTION_ARGS)
 									  quote_literal_cstr(createRoleUtilityQuery))));
 		}
 
-		CitusProcessUtility(parseTree,
-							createRoleUtilityQuery,
-							PROCESS_UTILITY_TOPLEVEL,
-							NULL,
-							None_Receiver, NULL);
+		ProcessUtilityForNode(parseTree,
+							  createRoleUtilityQuery,
+							  PROCESS_UTILITY_TOPLEVEL,
+							  NULL,
+							  None_Receiver, NULL);
```

Comment:
probably not related with this pr but I'm wondering why codecov claims that this is not covered